embedchain/apps/CustomApp.py

```diff
@@ -133,15 +133,18 @@ def _get_azure_openai_answer(prompt: str, config: ChatConfig) -> str:
         if config.top_p and config.top_p != 1:
             logging.warning("Config option `top_p` is not supported by this model.")
 
-        messages = CustomApp._get_messages(prompt)
+        messages = CustomApp._get_messages(prompt, system_prompt=config.system_prompt)
 
         return chat(messages).content
 
     @staticmethod
-    def _get_messages(prompt: str) -> List[BaseMessage]:
+    def _get_messages(prompt: str, system_prompt: Optional[str] = None) -> List[BaseMessage]:
         from langchain.schema import HumanMessage, SystemMessage
 
-        return [SystemMessage(content="You are a helpful assistant."), HumanMessage(content=prompt)]
+        return [
+            SystemMessage(content=system_prompt if system_prompt else "You are a helpful assistant."),
```

Comment:
We should only add if system prompt is present.
Can you update?

Reply:
fixed